frontend/cypress/integration/featureFiles/kiali_help.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Kiali help
+
+  User wants to login to Kiali and view kiali help like documentation, view bebug info, certificate information, about info
+
+  Scenario: Login to Kiali and Open Kiali home page
+    Given user opens base url
+    And user clicks Log In With OpenShift
+    And user clicks my_htpasswd_provider
+    And user fill in username and password
+    Then user see console in URL
```

Comment:
please refactor this scenario with 
`Given user is at administrator perspective`
for correct usage see
https://github.com/kiali/kiali/blob/bd662370f0950963d1151825915fc9e91db39474/frontend/cypress/integration/featureFiles/kiali_cookie.feature#L6